doc/userGuide.md

```diff
@@ -211,9 +211,13 @@ Usage : get.sh  --testdir|-t openjdktestdir
                 [--sdk_resource|-r ] : optional. Indicate where to download an sdk from - releases, nightly, upstream or customized
 
                 [--customizedURL|-c ] : optional. If downloading an sdk and if sdk source is set as customized, indicates sdk url 
-                [--openj9_repo ] : optional. OpenJ9 git repo. Default value: https://github.com/eclipse/openj9.git is used if not provided
-                [--openj9_sha ] : optional. OpenJ9 pull request sha.
-                [--openj9_branch ] : optional. OpenJ9 branch.
+                [--clone_openj9 ] : optional. ture or false. If set to true, get.sh will download openj9 repo functional tests. Default to false'
+                [--openj9_repo ] : optional. OpenJ9 git repo. Default value https://github.com/eclipse/openj9.git is used if not provided'
+                [--openj9_sha ] : optional. OpenJ9 pull request sha.'
+                [--openj9_branch ] : optional. OpenJ9 branch.'
+                [--tkg_repo ] : optional. TKG git repo. Default value https://github.com/AdoptOpenJDK/TKG.git is used if not provided'
+                [--tkg_sha ] : optional. TkG pull request sha.'
+                [--tkg_branch ] : optional. TKG branch.'
```

Comment:
- Default value master is used if not provided

- Please remove `'` at the end of each option.

Reply:
updated